### PR TITLE
K8s: Titan release notes

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Preview: 
https://redis.io/docs/staging/DOC-3665/operate/kubernetes/reference/supported_k8s_distributions/
https://redis.io/docs/staging/DOC-3665/operate/kubernetes/release-notes/7-4-2-12/

### CODE REVIEW

File: content/operate/kubernetes/release-notes/7-4-2-12.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+### New limitations
+
+- **Missing endpoint for admission endpoint (rare) (RED-119469)** Restart the operator pod.
+
+### Existing limitations
+
+- **The REDB “redisVersion” field can’t be used for memcached databases(RED-119152)**
+
+- **When modifying the database suffix for an Active-Active database, while the service-rigger is in a terminating state, the services-rigger will delete and create the ingress or route resources in a loop (RED-107687)** Wait until the services- rigger pod has finished to terminate it.
```
Comment (andy-stark-redis) [inline content/operate/kubernetes/release-notes/7-4-2-12.md:119]: Space between "services-" and "rigger" in the last occurrence.

**Review by andy-stark-redis — APPROVED** [review]
Can't verify the information but everything else looks good apart from maybe that one extra space.

File: content/operate/kubernetes/release-notes/7-4-2-12.md (hunk 1 above)
Comment (bascar) [reply content/operate/kubernetes/release-notes/7-4-2-12.md:119]: Highlights: This release has many enhancements, most notably support for persistent volume expansion in the REC. Also, some essential changes in module handling support newer capabilities prompted by changes in the underlying Redis Enterprise around versioning. 

Please change the legend at the top of the release notes and the supported distribution pages from "Deprecated" to "Deprecation warning". We changed this as well on RS for clarity.

The table in the release notes for the supported distributions seems to be misalgned on the K8s version and the columns below.

**Review by bascar — APPROVED** [review]
Approved with noted changes or updates.

**Review by bascar — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**